skills/lism-mock-guide/SKILL.md

```
@@ -0,0 +1,147 @@
+---
+name: lism-mock-guide
+description: 'デザインモックを@lism-css/mockのデータディレクトリ（pages/*.jsx|tsx・tokens.json・mock.config.json）として作成・編集する時の作業ガイド。lism-mock init→実装→lism-mock checkの自己検証ワークフローと、ページID・import許可リスト・トークン上書きのデータ契約に従う。マークアップの書き方自体はlism-css-guideを併用する。lism-mock / @lism-css/mock / デザインモックの作成・修正で使う。'
+---
+
+# Lism CSSデザインモック作成ガイド（@lism-css/mock）
+
+デザインモックを`@lism-css/mock`の**データディレクトリ**（1画面=1ファイルの`pages/`＋小さな設定ファイル）として組むためのガイドです。プレビューアプリ（ビューア）はCLIに同梱されており、モック側にビルド設定や`package.json`は不要です。このスキルが扱うのは「モックとしてどんなファイルを置けるか（データ契約）」と「initからcheckまでのワークフロー」だけです。
+
+> **前提（必須）: `lism-css-guide`を併用する。**
+> このスキルは`lism-css-guide`が同じ階層に導入されていることを前提にします。マークアップ自体の書き方 — Primitive選定・トークン照合・Property Class・レスポンシブ設計 — はすべて`lism-css-guide`の実装フローに従ってください。無い場合は、`lism-cli skill add`（引数なしで全スキル一括導入）での追加をユーザーに案内してください。guideなしで、推測だけでモックを書かないでください。
+
```

markdownlint-cli2 (0.23.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)